src/Kernels/Local.h

```diff
@@ -53,6 +54,9 @@ namespace seissol::kernels {
 }
 
 class seissol::kernels::Local : public LocalBase {
+  protected:
+    seissol::initializer::parameters::DamagedElasticParameters m_damagedElasticParameters;
```

Comment:
Do you really want to store a copy here?

Reply:
Modified to a pointer